activiti-cloud-services-runtime-bundle/activiti-cloud-services-core/src/test/resources/test-process-diagram.properties

```diff
@@ -0,0 +1,2 @@
+activiti.engine.diagram.activity.font=Lucida
```

Comment:
@constantin-ciobotaru Are these the default values? I would like to avoid users need to set up this thing.